packages/web/pages/submission/[step].tsx

```diff
@@ -0,0 +1,46 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import { Form, Stepper } from '@components';
+
+const FORM_STEPS = ['Primary', 'Contact', 'Credential', 'Preferences', 'Review and Submit'];
+
+const Submission = () => {
+  const router = useRouter();
+
+  const step = Number(router.query.step);
+
+  return (
+    <Form>
+      <div className='w-full p-4 border-b'>
+        <Stepper formSteps={FORM_STEPS} step={step} />
```

Comment:
Stepper is not interactive ?

Reply:
I think that was the outcome of our discussions, to keep it like Imms form for simplicity for now

Reply:
OK